### ISSUE TITLE
Ugly carousel images while loading index

### ISSUE THREAD

**Issue by singhpratyush** [iiitv/Odyssy#69]
![screenshot from 2017-06-11 19 33 33](https://user-images.githubusercontent.com/10860278/27011622-1dad3d60-4edd-11e7-8ff8-7f999abff7f2.png)

**Comment by geekyJock8** [comment]
I tried to correct it by initially setting the display of class carousel to none and by using Jquery's show() function. But the Jquery is not working, even after the page is completely loaded, the carousel is invisible.

**Comment by Monal5031** [comment]
@geekyJock8 It is because the js file is added but not linked here https://github.com/iiitv/Odyssy/blob/master/wsgi/odyssy/basic/templates/basic/base.html#L185

**Comment by geekyJock8** [comment]
But rest of the code js file is working. I can confirm this because carousel's script is being loaded from the file. And on modifying the code carousel gets modified.

**Comment by Monal5031** [comment]
I updated the link locally and its working now @geekyJock8

**Comment by Monal5031** [comment]
Also @geekyJock8 There are two js files there, only one is linked `main.js` is linked 😄

**Comment by geekyJock8** [comment]
Changing to {% static 'basic/js/jquery.paginate.min.js' %} didn't really solve the problem. But I was able to correct this by making a div inside the for loop of the carousel and using show() on that div. So this ugly carousel problem is solved but there's a new problem. The slick carousel is acting weird, it's showing empty space after the last photo in the carousel. 
![image](https://user-images.githubusercontent.com/19239291/27679746-388d7d54-5cd7-11e7-9ae4-7b5642fc287b.png)

Basically, it's showing 3 extra empty photos, after scrolling through these empty photos it goes back to the initial image. To correct this I tried to turn off the infinite scrolling of Slick but on doing so the entire carousel vanished. Please help @ansarimofid

**Comment by singhpratyush** [comment]
I think this can be fixed by placing the `js` for Slick at the top in HTML.

**Comment by Monal5031** [comment]
@singhpratyush Shall I try it in my PR we can see if it will work or not.
